### PR TITLE
Reintroduce pijul (at 0.3.0)

### PR DESCRIPTION
**Pull request by FlorentBecker**
###### Motivation for this change
pijul has just had its first public release

###### Things done

- [ ] Tested using sandboxing
  ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS,
    or option `build-use-sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file)
    on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] Linux
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

File: pkgs/applications/version-management/pijul/default.nix [hunk 1]
```diff
@@ -0,0 +1,29 @@
+{ stdenv, fetchurl, rustPlatform, perl }:
+
+with rustPlatform;
+
+buildRustPackage rec {
+  name = "pijul-${version}";
+  version = "0.3.0";
+
+  src = fetchurl {
+    url = https://pijul.org/releases/pijul-0.3.tar.gz;
```
Comment (Mic92) [inline pkgs/applications/version-management/pijul/default.nix]:
```
url = "https://pijul.org/releases/pijul-${version}.tar.gz";
```

Comment (domenkozar) [inline pkgs/applications/version-management/pijul/default.nix]: Then version should be 0.3 (without ending zero)

**Comment by vbgl** [comment]
On Darwin, the `Security` framework is needed:

```
diff --git a/pkgs/applications/version-management/pijul/default.nix b/pkgs/applications/version-management/pijul/default.nix
index 1b1a54da00..cef0f23f91 100644
--- a/pkgs/applications/version-management/pijul/default.nix
+++ b/pkgs/applications/version-management/pijul/default.nix
@@ -1,4 +1,4 @@
-{ stdenv, fetchurl, rustPlatform, perl }:
+{ stdenv, darwin, fetchurl, rustPlatform, perl }:
 
 with rustPlatform;
 
@@ -13,7 +13,10 @@ buildRustPackage rec {
 
   sourceRoot = "pijul/pijul";
 
-  buildInputs = [ perl ];
+  buildInputs = [ perl ]
+  ++ stdenv.lib.optionals stdenv.isDarwin
+  (with darwin.apple_sdk.frameworks; [ Security ])
+  ;
 
   doCheck = false;
   
```

**Comment by copumpkin** [comment]
PIJUL RETURNS!!

**Pull request merged**